src/server/api/geo/getForestAgreementLayer.ts

```diff
@@ -1,19 +1,39 @@
 import { Request, Response } from 'express'
 
 import { CountryIso } from '@meta/area'
+import { ForestSource } from '@meta/geo'
 
 import { GeoController } from '@server/controller/geo'
 import Requests from '@server/utils/requests'
 
 export const getForestAgreementLayer = async (req: Request, res: Response) => {
   try {
     const countryIso: CountryIso = req.params.countryIso as CountryIso
-    const gteAgreementLevel = Number(req.params.gteAgreementLevel)
-    const gteHansenTreeCoverPerc = Number(req.params.gteHansenTreeCoverPerc)
+    const gteAgreementLevel = Number(req.query.gteAgreementLevel)
+    const gteHansenTreeCoverPerc = Number(req.query.gteHansenTreeCoverPerc)
 
```

Comment:
to avoid casting to all query parameters, you could type the Request.
e.g.
```typescript
export const getForestAgreementLayer = async (req: Request<{countryIso:CountryIso}, never, never, {gteAgreementLevel:number;gteHansenTreeCoverPerc:number; layer:Array<ForestSource>}>
, res: Response) => {
const {countryIso}=req.params
const {gteAgreementLevel, gteHansenTreeCoverPerc,layer}=req.query
...
```

Btw we're migrating apis to use only query, not params. we can discuss this later on. not important at all now